### PR TITLE
Refactor ShowTab to take into account NetTab.isPopOut

### LINKED ISSUE #1533: Refactor how popup tabs are checked B: 25
# Description
New popout tabs have to be placed into a hardcoded check to make them popout.

https://github.com/unitystation/unitystation/blob/f9c5597aac500808ad4d67615d89a69f6caab47c/UnityProject/Assets/Scripts/UI/ControlTabs.cs#L426

This isn't ideal because the tabs already have a `IsPopOut` variable to determine this,

image

The code should be refactored so this hardcoded check isn't necessary anymore.

## Category
TODO

### PR DESCRIPTION
**Pull request by hecksadecimal**
### Purpose
Fixes #1533 

ShowTab used to rely on a static list of tab types, it's been refactored to instead rely on the tab's isPopout variable.

### Open Questions and Pre-Merge TODOs

- [x]  This fix is tested on the same branch it is PR'ed to.
- [x]  I correctly commented my code
- [x]  My code is indented with tabs and not spaces
- [x]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [x]  This PR does not bring up any new compile errors
- [x]  This PR has been tested in editor
- [x]  This PR has been tested in multiplayer

### Notes:
If there's a better way to do this please let me know and I'll work on changing it. I'm not too familiar with Unity just yet.

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/ControlTabs.cs [hunk 1]
```diff
@@ -420,13 +420,10 @@ public static void ShowTab(NetTabType type, GameObject tabProvider, ElementValue
 			return;
 		}
 
-		bool isPopOut = false;
+		NetTab thisTab = NetworkTabManager.Instance.Get(tabProvider, type);
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/UI/ControlTabs.cs]: Won't work for client – NetworkTabManager is serverside-only:
https://github.com/unitystation/unitystation/blob/c29f55cf197f2d2a2e8b56ac77765c6c0327cc37/UnityProject/Assets/Scripts/Managers/NetworkManagement/NetworkTabManager.cs#L7

**Review by fomalsd — CHANGES_REQUESTED** [review]
You're trying to access a tab list that's only present on server.
Try getting isPopOut property after you actually spawned it locally

File: UnityProject/Assets/Scripts/UI/ControlTabs.cs (hunk 1 above)
Comment (hecksadecimal) [reply UnityProject/Assets/Scripts/UI/ControlTabs.cs]: > 
> 
> You're trying to access a tab list that's only present on server.
> Try getting isPopOut property after you actually spawned it locally

How are these latest changes?

**Comment by hecksadecimal** [comment]
Made changes based on the input given.

**Review by fomalsd — APPROVED** [review]

**Comment by fomalsd** [comment]
Good job!

**Pull request merged**